src/plugins/saved_objects_management/public/lib/index.ts

```diff
@@ -8,7 +8,6 @@
 
 export { fetchExportByTypeAndSearch } from './fetch_export_by_type_and_search';
 export { fetchExportObjects } from './fetch_export_objects';
-export { canViewInApp } from './in_app_url';
```

Comment:
Replaced by [class method](https://github.com/elastic/kibana/blob/master/src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx#L88-L93) in https://github.com/elastic/kibana/pull/112034